README.md

```diff
@@ -70,8 +70,7 @@ Wagtail is used by [NASA](https://www.nasa.gov/), [Google](https://www.google.co
 
 ### 📖 Documentation
 
-[docs.wagtail.org](https://docs.wagtail.org/) is the full reference for Wagtail, and includes guides for developers, designers and editors, alongside [release notes](https://docs.wagtail.org/en/stable/releases/) and our [roadmap](https://github.com/wagtail/wagtail/wiki/Roadmap).
-
+[docs.wagtail.org](https://docs.wagtail.org/) is the full reference for Wagtail, and includes guides for developers, designers and editors, alongside [release notes](https://docs.wagtail.org/en/stable/releases/) and our [roadmap](https://wagtail.org/roadmap/).
```

Comment:
We now put the editor guide in guide.wagtail.org, so the first part of the sentence isn’t completely accurate anymore.

Reply:
ohh , okay thank @laymonage , so whats the accurate one ??

Reply:
@Rohit-Singh-01 maybe something like this would be more suitable. Feel free to raise a new PR. Remember to sync your repo with the latest `main` before you create a new branch though. 

```md
[docs.wagtail.org](https://docs.wagtail.org/) is the full reference for Wagtail, including guides for developers and designers plus [release notes](https://docs.wagtail.org/en/stable/releases/). The official [roadmap](https://wagtail.org/roadmap/) contains future features and enhancements we have planned.

**For editors** who want to understand how to manage content within the Wagtail admin interface, see the [Wagtail User Guide](https://guide.wagtail.org/).

```

Feel free to refine, maybe pass the whole 'documentation' section through ChatGPT for simple readability improvements, but minimal changes are recommended. We don't want to rewrite things needlessly.